src/units/unit.hpp

```diff
@@ -1559,6 +1559,18 @@ class unit : public std::enable_shared_from_this<unit>
 		return halo_.value_or("");
 	}
 
+	const std::vector<std::string> halo_or_icon_abilities(const std::string& image_type, const map_location& loc) const;
```

Comment:
Does anything use this form of the method to pass in another location? It looks like the only two callers just use the single argument version, which just uses `loc_`.

Reply:
changed in https://github.com/wesnoth/wesnoth/pull/7856/commits/8f83bfab42ffb702d08c1a710d464143b51ee8f3